### PR TITLE
Remote control: state is saved now.

### LINKED ISSUE #350: Feature Request: save state of "Remote control"
Hello

Please, add status of "Remote control" into configure file

### PR DESCRIPTION
**Pull request by kukabu**
Fixes #350.

### CODE REVIEW

File: src/applications/gqrx/remote_control.cpp [hunk 1]
```diff
@@ -74,35 +74,55 @@ void RemoteControl::stop_server()
 /*! \brief Read settings. */
 void RemoteControl::readSettings(QSettings *settings)
 {
-    bool conv_ok;
+    if (!settings)
+        return;
 
-    rc_freq = settings->value("input/frequency", 144500000).toLongLong(&conv_ok);
-    rc_filter_offset = settings->value("receiver/offset", 0).toInt(&conv_ok);
```
Comment (csete) [inline src/applications/gqrx/remote_control.cpp]: Why are you deleting these?

File: src/applications/gqrx/remote_control.cpp [hunk 2]
```diff
@@ -74,35 +74,55 @@ void RemoteControl::stop_server()
...
+    settings->beginGroup("remote_control");
```
Comment (csete) [inline src/applications/gqrx/remote_control.cpp]: Although the beginGroup() / endGroup() change is useful, they don't have anything to do with the topic and should therefore be a separate commit and pull request. Including them here only makes the present pull request more difficult to understand.

File: src/applications/gqrx/remote_control.cpp (hunk 1 above)
Comment (kukabu) [inline src/applications/gqrx/remote_control.cpp]: it's unnecessary when RC server running we get the current value from main code.

File: src/applications/gqrx/remote_control.cpp (hunk 2 above)
Comment (kukabu) [inline src/applications/gqrx/remote_control.cpp]: ok, I will separate commits.

**Comment by kukabu** [comment]
ok, I try to find more simple way to start RC in main code.

**Comment by kukabu** [comment]
@csete, is it ok?

**Comment by csete** [comment]
I will take a look at it after v2.6 is out, hopefully by the end of this
week.

**Comment by csete** [comment]
Looks fine, lets use this.

**Comment by kukabu** [comment]
Thanks!
